src/mesido/head_loss_class.py

```diff
@@ -768,6 +768,9 @@ def _hn_pipe_head_loss(
                 else:
                     big_m_lin = big_m
                     constraint_nominal = (constraint_nominal * big_m_lin) ** 0.5
```

Comment:
1.  constraint_nominal is calculated twice, one if these calcs have to be deleted
2. Currently the calculation "np.abs(head_loss_nominal * (a_vec * q_nominal + b_vec)" occurs twice. Once here and also in line 761. I would rather create an additional variable new_var = np.abs(head_loss_nominal * (a_vec * q_nominal + b_vec) and then use this new_var in both calculation. This is similar to the old code where constraint_nominal was used in line 770 instead of redoing the calculation of the variables

Reply:
updated